src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.cs

```diff
@@ -739,9 +746,9 @@ public void OnTrailersComplete()
                 }
                 else if (!HasResponseStarted)
                 {
-                    // If the request was aborted and no response was sent, there's no
-                    // meaningful status code to log.
-                    StatusCode = 0;
+                    // If the request was aborted and no response was sent, we use status code 499 for logging
+                    // https://learn.microsoft.com/en-us/azure/application-gateway/http-response-codes#499--client-closed-the-connection
```

Comment:
This link isn't authoritative and is only indirectly related to our decision. The comment is adequate. 
```suggestion
```